modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1786 @@
+Added 1 constraint.
+----
+
+======
+
+.Create a relationship property type constraint with a union type
+======
+
+.Create a constraint requiring `tags` properties on `PART_OF` relationships to either of type `STRING` or `LIST<STRING NOT NULL>`
+[source, cypher]
+----
+CREATE CONSTRAINT part_of_tags
+FOR ()-[part:PART_OF]-() REQUIRE part.tags IS :: STRING | LIST<STRING NOT NULL>
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[type-constraints-allowed-properties]]
+==== Allowed types
+
+The allowed property types for property type constraints are:
+
+* `BOOLEAN`
+* `STRING`
+* `INTEGER`
+* `FLOAT`
+* `DATE`
+* `LOCAL TIME`
+* `ZONED TIME`
+* `LOCAL DATETIME`
+* `ZONED DATETIME`
+* `DURATION`
+* `POINT`
+* `LIST<BOOLEAN NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<STRING NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<INTEGER NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<FLOAT NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<DATE NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<LOCAL TIME NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<ZONED TIME NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<LOCAL DATETIME NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<ZONED DATETIME NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<DURATION NOT NULL>` label:new[Introduced in 5.10]
+* `LIST<POINT NOT NULL>` label:new[Introduced in 5.10]
+* Any closed dynamic union of the above types, e.g. `INTEGER | FLOAT | STRING`. label:new[Introduced in 5.11]
+
+For a complete reference describing all types available in Cypher, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
+
+
+[[create-property-type-constraint-compliant-data]]
+==== Create data that complies with existing property type constraints
+
+.Create a node that complies with existing node property type constraint
+======
+
+.Create an `Movie` node with a `STRING` `title` property
+[source, cypher]
+----
+CREATE (movie:Movie {title:'Iron Man'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 label, created 1 node, set 1 properties
+----
+
+======
+
+.Create a relationship that complies with existing relationship property type constraint
+======
+
+.Create a `PART_OF` relationship with an `INTEGER` `order` property
+[source, cypher]
+----
+MATCH (movie:Movie {title:'Iron Man'})
+CREATE (movie)-[part:PART_OF {order: 3}]->(franchise:Franchise {name:'MCU'})
+----
+
+.Result
+[queryresult]
+----
+Added 1 label, added 1 node, created 1 relationship, set 2 properties
+----
+
+======
+
+
+
+[role=label--enterprise-edition]
+[[create-key-constraints]]
+=== Create key constraints
+
+Key constraints ensure that all properties exist and that the combined property values are unique for all nodes with a specific label or all relationships with a specific type.
+For composite property uniqueness constraints on multiple properties, it is the combination of property values that must be unique.
```

Comment:
```suggestion
Key constraints ensure that all properties exist and that the property value (for single property constraints) or combination of property values (for composite constraints) are unique for all nodes with a specific label or all relationships with a specific type.
```
Realized that the first sentence also said `combined`, so might be better to combine them.
Or
```suggestion
Key constraints ensure that the property exist and the property value is unique for all nodes with a specific label or all relationships with a specific type.
For composite key constraints on multiple properties, all properties must exists and the combination of property values must be unique.
```
if we still want it as two sentences maybe?